Testing/Unit/Rego/Defender/DefenderConfig_01_test.rego

```diff
@@ -1,185 +1,171 @@
-package defender
+package defender_test
 import future.keywords
+import data.defender
+import data.report.utils.ReportDetailsBoolean
+
+
+CorrectTestResult(PolicyId, Output, ReportDetailString) := true if {
+    RuleOutput := [Result | some Result in Output; Result.PolicyId == PolicyId]
+
+    count(RuleOutput) == 1
+    RuleOutput[0].RequirementMet == true
+    RuleOutput[0].ReportDetails == ReportDetailString
+} else := false
+
+IncorrectTestResult(PolicyId, Output, ReportDetailString) := true if {
+    RuleOutput := [Result | some Result in Output; Result.PolicyId == PolicyId]
+
+    count(RuleOutput) == 1
+    RuleOutput[0].RequirementMet == false
+    RuleOutput[0].ReportDetails == ReportDetailString
+} else := false
+
+FAIL := ReportDetailsBoolean(false)
+
+PASS := ReportDetailsBoolean(true)
+
 
 #
 # Policy 1
 #--
```

Comment:
Not sure what this adds after our reformatting policy ids.

Reply:
This is part of the style  guide to split test cases. I find it incredibly useful for files with many test cases for a policy

Reply:
My thought was instead of Policy 1 it would be more meaningful as MS.Defender.1.1v1 instead of Policy 1